modules/arcgis/package.json

```diff
@@ -40,7 +40,7 @@
     "esri-loader": "^3.3.0"
   },
   "peerDependencies": {
-    "@arcgis/core": "^4.0.0",
+    "@arcgis/core": "^4.28.0",
```

Comment:
Is there anything specifically depending on a feature in this version? Better keep peer dependency restrictions loose for best compatibility.

Reply:
At the beginning of this work it appeared that an update would be required in order to use the webgl2 context. However, that was before the context was attached to the device. It seems to work fine if I downgrade at this point.